content/applications/inventory_and_mrp/purchase/products/temporary_reordering.rst

```diff
@@ -0,0 +1,120 @@
+==========================
+Temporary reordering rules
+==========================
+
+Some businesses require certain products to always have a minimum quantity of stock on-hand at any
+given time. To avoid stock falling below a certain threshold, companies can create *reordering
+rules* in Odoo to automate purchase orders for specific products.
+
+Reordering rules keep the forecasted stock levels above a certain threshold without exceeding a
+specified upper limit. When a product has a reordering rule applied to it, and that product's
+quantity falls below a specified number, Odoo uses the specified *route* on the product form
+(e.g. *Buy* or *Manufacture*), and creates an order to replenish the stock.
+
+However, there are some instances in which businesses might not want or need specific products to be
+replenished automatically. In such cases, they can rely on "temporary" reordering rules.
+
+In Odoo, when a product has 0 quantity in stock, has a *Buy* route selected, and is included in a
+sales order (SO), Odoo creates a "temporary" reordering rule in the replenishment dashboard for that
+product, which is deleted upon confirmation of the purchase order (PO) created for that product.
+
+.. seealso::
+   - :doc:`../../inventory/product_management/product_replenishment/reordering_rules`
+   - :doc:`../../purchase/products/reordering`
+
+Configuration
+=============
+
+To trigger temporary reordering rules once its stock reaches 0, a product needs to be configured in
+a specific way.
+
+To create a product that will trigger temporary reordering rules, go to :menuselection:`Inventory
+app --> Products --> Products`, and click :guilabel:`New`.
+
+On the product form, enter a name for the product, and enable selling and reordering by checking the
+:guilabel:`Can be Sold` and :guilabel:`Can be Purchased` options underneath the :guilabel:`Product
+Name` field. Finally, set the :guilabel:`Product Type` to `Storable Product`, under the
+:guilabel:`General Information` tab.
```

Comment:
since "can be sold" and "can be purchased" are enabled by default for products created in the inventory app, let's make this part a little shorter (:
```suggestion
On the product form, enter the product name, and ensure the :guilabel:`Can be Sold` and
:guilabel:`Can be Purchased` options are enabled underneath the :guilabel:`Product Name` field.
Finally, set the :guilabel:`Product Type` to `Storable Product`, under the :guilabel:`General
Information` tab.
```